### DISCUSSION TITLE
Bring back click to go to article

### DISCUSSION THREAD

**Discussion by ekaralar** [dailydotdev/daily#669] [Feature Requests]
Hello,
First, thanks for the great product. I'm very used to clicking on an article tile and going to it directly. I honestly do not want to see the "summary" page. 
Could you provide an option to just click on the tile and go to the page directly?

(@idoshamun: Thanks for providing the main discussion thread, feel free to delete this one)

Thanks,

**Comment by idoshamun** [comment]
Hi @ekaralar,

I provided an answer for this here: https://github.com/dailydotdev/daily/discussions/653.
Let's keep the discussion going there :)
